templates/base.html

```diff
@@ -137,7 +137,46 @@
         <script src="//cdn.ravenjs.com/3.0.4/raven.min.js"></script>
         <script>Raven.config('{% sentry_public_dsn %}').install()</script>
         {% block script %}{% endblock %}
-
+        <!-- Piwik -->
+<script type="text/javascript">
+    var _paq = _paq || [];
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+        var u="//analytics.organize.org/";
+        _paq.push(['setTrackerUrl', u+'piwik.php']);
+        _paq.push(['setSiteId', '1']);
+        if (gender != '') {
```

Comment:
@nasief This will not work, here you are just referencing gender and birhthdate without getting it from anywhere. Please refer to the code below for how we are getting those values for GA usage.